src/components/footer/footer.scss

```diff
@@ -1,43 +1,67 @@
 @import '../bootstrap/bootstrap';
 
+$social-icon-size: 2.5rem;
+
 .footer {
   background: $darkgrey;
   color: $white;
 }
 
-.logo path {
+.footer .logo path {
   fill: $gold;
 }
 
-.logo svg {
-  margin-left: 2rem;
+.footer .logo svg {
   padding: 1rem 0;
   height: 7rem;
 }
 
 .footer-top {
+  padding-left: 2rem;
+  padding-right: 2rem;
   border-spacing: 15px;
+  justify-content: space-between;
 }
 
 .footer-hr-container {
-  padding: 0 !important;
-}
-
-.footer.row > div {
   padding: 0;
 }
 
 .footer hr {
-  margin: 0rem 1.66667rem 0rem 2rem !important;
+  margin: 0rem 1.66667rem 0rem 2rem;
```

Comment:
another nitpick that I should incorporate into a linter, but technically `0` value units don't need `rem`,`px`, etc

Reply:
Similar to the previous comment, I left this as part of the existing codebase. I will go ahead and make this change as well